From: Juergen Borleis <jbe@pengutronix.de>
Date: Fri, 4 Dec 2015 14:53:05 +0100
Subject: [PATCH] DTS/RTC: remove a no longer used property

From the schematics there are no external capacitors to trim the crystal,
so the default settings for this device should be correct.

Signed-off-by: Juergen Borleis <jbe@pengutronix.de>
---
 arch/arm/boot/dts/imx6qdl-cpu.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6qdl-cpu.dtsi b/arch/arm/boot/dts/imx6qdl-cpu.dtsi
index 5186ec931aa2..739fc6c487df 100644
--- a/arch/arm/boot/dts/imx6qdl-cpu.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-cpu.dtsi
@@ -338,7 +338,6 @@
 	i2c_rtc: pcf85063@51 {
 		compatible = "nxp,pcf85063";
 		reg = <0x51>;
-		quartz_load = "12.5pF";
 	};
 };
 
